watchtower/__init__.py

```diff
@@ -219,7 +292,7 @@ def emit(self, message):
         if stream_name not in self.sequence_tokens:
             self.sequence_tokens[stream_name] = None
 
-        if isinstance(message.msg, Mapping):
+        if self.json_serialize_default and isinstance(message.msg, Mapping):
```

Comment:
Dumping JSON will only happen once so it is fine to leave this here instead of removing it.